### PR TITLE
bump tidb version on tiup docs

### PR DESCRIPTION
**Pull request by nexustar**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Bump TiDB version to v7.1.0

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13711
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -27,17 +27,17 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
 
 > **注意：**
 >
-> 如果原集群是 3.0 或 3.1 或更早的版本，不支持直接升级到 7.0.0 及后续修订版本。你需要先从早期版本升级到 4.0 后，再从 4.0 升级到 7.0.0 及后续修订版本。
+> 如果原集群是 3.0 或 3.1 或更早的版本，不支持直接升级到v7.1.0 及后续修订版本。你需要先从早期版本升级到 4.0 后，再从 4.0 升级到v7.1.0 及后续修订版本。
```
Comment (Oreoxmt) [inline upgrade-tidb-using-tiup.md:30]:
```suggestion
> 如果原集群是 3.0 或 3.1 或更早的版本，不支持直接升级到 v7.1.0 及后续修订版本。你需要先从早期版本升级到 4.0 后，再从 4.0 升级到 v7.1.0 及后续修订版本。
```

File: upgrade-tidb-using-tiup.md [hunk 2]
```diff
@@ -27,17 +27,17 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
 
 ## 1. 升级兼容性说明
 
 - TiDB 目前暂不支持版本降级或升级后回退。
-- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 7.0.0 版本。
-- 若要将 3.0 之前的版本升级至 7.0.0 版本：
+- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 7.1.0 版本。
```
Comment (Oreoxmt) [inline upgrade-tidb-using-tiup.md:35]:
```suggestion
- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 v7.1.0 版本。
```

File: upgrade-tidb-using-tiup.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ tiup update cluster
 
 > **注意：**
 >
-> 升级到 7.0.0 版本前，请确认已在 4.0 修改的参数在 7.0.0 版本中是兼容的，可参考 [TiKV 配置文件描述](/tikv-configuration-file.md)。
+> 升级到v7.1.0 版本前，请确认已在 4.0 修改的参数在v7.1.0 版本中是兼容的，可参考 [TiKV 配置文件描述](/tikv-configuration-file.md)。
```
Comment (Oreoxmt) [inline upgrade-tidb-using-tiup.md:145]:
```suggestion
> 升级到 v7.1.0 版本前，请确认已在 4.0 修改的参数在 v7.1.0 版本中是兼容的，可参考 [TiKV 配置文件描述](/tikv-configuration-file.md)。
```

File: upgrade-tidb-using-tiup.md [hunk 4]
```diff
@@ -179,12 +179,12 @@ tiup cluster check <cluster-name> --cluster
 tiup cluster upgrade <cluster-name> <version>
 ```
 
-以升级到 7.0.0 版本为例：
+以升级到v7.1.0 版本为例：
```
Comment (Oreoxmt) [inline upgrade-tidb-using-tiup.md:182]:
```suggestion
以升级到 v7.1.0 版本为例：
```

File: tiup/tiup-cluster.md [hunk 5]
```diff
@@ -555,11 +555,11 @@ tiup cluster audit
 Starting component `cluster`: /home/tidb/.tiup/components/cluster/v1.11.3/cluster audit
 ID      Time                       Command
 --      ----                       -------
-4BLhr0  2022-03-29T23:55:09+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.0.0 /tmp/topology.yaml
-4BKWjF  2022-03-029T23:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.0.0 /tmp/topology.yaml
-4BKVwH  2022-03-29T23:02:08+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.0.0 /tmp/topology.yaml
+4BLhr0  2022-03-29T23:55:09+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.1.0 /tmp/topology.yaml
+4BKWjF  2022-03-029T23:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.1.0 /tmp/topology.yaml
+4BKVwH  2022-03-29T23:02:08+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.1.0 /tmp/topology.yaml
 4BKKH1  2022-03-29T16:39:04+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster destroy test
```
Comment (Oreoxmt) [inline tiup/tiup-cluster.md:561]:
```suggestion
4BLhr0  2023-05-31T23:55:09+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.1.0 /tmp/topology.yaml
4BKWjF  2023-05-31T23:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.1.0 /tmp/topology.yaml
4BKVwH  2023-05-31T23:02:08+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.1.0 /tmp/topology.yaml
4BKKH1  2023-05-31T16:39:04+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster destroy test
```

File: tiup/tiup-cluster.md [hunk 6]
```diff
@@ -555,11 +555,11 @@ tiup cluster audit
...
-4BKKDx  2022-03-29T16:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.0.0 /tmp/topology.yaml
+4BKKDx  2022-03-29T16:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.1.0 /tmp/topology.yaml
```
Comment (Oreoxmt) [inline tiup/tiup-cluster.md:562]:
```suggestion
4BKKDx  2023-05-31T16:36:57+08:00  /home/tidb/.tiup/components/cluster/v1.11.3/cluster deploy test v7.1.0 /tmp/topology.yaml
```

**Review by Oreoxmt — APPROVED** [review]

File: upgrade-tidb-using-tiup.md [hunk 7]
```diff
@@ -49,7 +49,7 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
 
 ### 2.1 查阅兼容性变更
 
-查阅 TiDB v7.0.0 release notes 中的[兼容性变更](/releases/release-7.0.0.md#兼容性变更)。如果有任何变更影响到了你的升级，请采取相应的措施。
+查阅 TiDB v7.1.0 release notes 中的[兼容性变更](/releases/release-7.0.0.md#兼容性变更)。如果有任何变更影响到了你的升级，请采取相应的措施。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:52]:
```suggestion
查阅 TiDB v7.1.0 release notes 中的[兼容性变更](/releases/release-7.1.0.md#兼容性变更)。如果有任何变更影响到了你的升级，请采取相应的措施。
```

Comment (qiancai) [reply upgrade-tidb-using-tiup.md:52]: 这个 PR 需要等到 release notes PR 合并后再合并

File: upgrade-tidb-using-tiup.md [hunk 8]
```diff
@@ -27,17 +27,17 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
 
 > **注意：**
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:28]: L12 ~ L18 需要改成
- 使用 TiUP 从 TiDB 6.0-6.4 版本升级至 TiDB 6.1。
- 使用 TiUP 从 TiDB 7.0 版本升级至 TiDB 6.1。

Comment (Oreoxmt) [reply upgrade-tidb-using-tiup.md:28]: Done https://github.com/pingcap/docs-cn/pull/14077/commits/af37080f9ee3ce483352921a5052ad01f2baa82a

File: upgrade-tidb-using-tiup.md [hunk 9]
```diff
@@ -7,15 +7,15 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
-- 使用 TiUP 从 TiDB 6.6 版本升级至 TiDB 7.0。
+- 使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 7.1。
+- 使用 TiUP 从 TiDB 5.0-5.4 版本升级至 TiDB 7.1。
+- 使用 TiUP 从 TiDB 6.0 版本升级至 TiDB 7.1。
+- 使用 TiUP 从 TiDB 6.1 版本升级至 TiDB 7.1。
+- 使用 TiUP 从 TiDB 6.2 版本升级至 TiDB 7.1。
+- 使用 TiUP 从 TiDB 6.3 版本升级至 TiDB 7.1。
+- 使用 TiUP 从 TiDB 6.4 版本升级至 TiDB 7.1。
+- 使用 TiUP 从 TiDB 6.5 版本升级至 TiDB 7.1。
+- 使用 TiUP 从 TiDB 6.6 版本升级至 TiDB 7.1。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:18]:
```suggestion
- 使用 TiUP 从 TiDB 6.0-6.6 版本升级至 TiDB 7.1。
- 使用 TiUP 从 TiDB 7.0 版本升级至 TiDB 7.1。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**